content/events/1v1-might.md

```diff
@@ -9,19 +9,23 @@ Divisions of up to 6 players, each playing one 1v1 match, once as host and once
 - Each competitor will play two 1v1 might matches vs the other players in their division, once as host/home and once as away
 - In the event of a tie (drawn), best two of three 1v1 matches as a run off
 - Polympics time out rules
-- Medium/regular map size. Might. 24 hour turns
-- v45 Tribe tier list. Host choses map type and tribe tier -- see tribe tier list below
+- Medium or large map size. 24 hour turns
+- Host choses map type. Waterworld and archipelagos maps are banned
 - One t2 restart per player per match
+- Tribe tiers will be assigned for each division
 
 ### v-45 tribe tier list
-- 1: C 
-- 2: ZKB 
-- 3: EYLI 
-- 4: PAiQX 
-
+- Tier 1: ZKB 
+- Tier 2: EYLI 
+- Tier 3: PAiQX 
+- Tier 4: VHAqO
+ 
 ## Notes
 
 - Teams may send multiple competitors to this event
+- Cymanti banned
 - No bots are allowed
 - Mobile only
-
+- Hosts should disable all banned tribes before sending game invite
+- Friend requests should be exchanged and games started within 24 hours of match assignments
+- Failure to exchance friend requests, start matches, or otherwise follow the rules are grounds for removal from the event at commissioner discretion
```

Comment:
```suggestion
- Failure to exchange friend requests, start matches, or otherwise follow the rules are grounds for removal from the event at commissioner discretion
```
Typo